WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -0,0 +1,685 @@
+import Foundation
+import WordPressComAnalytics
+import WordPressComStatsiOS
+import WordPressShared
+
+@objc class PostListViewController : AbstractPostListViewController, UIViewControllerRestoration, PostCardTableViewCellDelegate {
+    
+    static private let postCardTextCellIdentifier = "PostCardTextCellIdentifier"
+    static private let postCardImageCellIdentifier = "PostCardImageCellIdentifier"
+    static private let postCardRestoreCellIdentifier = "PostCardRestoreCellIdentifier"
+    static private let postCardTextCellNibName = "PostCardTextCell"
+    static private let postCardImageCellNibName = "PostCardImageCell"
+    static private let postCardRestoreCellNibName = "RestorePostTableViewCell"
+    static private let postsViewControllerRestorationKey = "PostsViewControllerRestorationKey"
+    static private let statsStoryboardName = "SiteStats"
+    static private let currentPostListStatusFilterKey = "CurrentPostListStatusFilterKey"
+    static private let currentPostAuthorFilterKey = "CurrentPostAuthorFilterKey"
+    
+    // TODO: low cap on first char!
+    
+    static private let statsCacheInterval = NSTimeInterval(300) // 5 minutes
+    static private let postCardEstimatedRowHeight = Float(100.0)
+    static private let postCardRestoreCellRowHeight = Float(54.0)
```

Comment:
Maybe just store these two constants as `CGFloat` instead of `Float`, since they're casted anyways whenever they're used